index.html

```diff
@@ -286,6 +286,42 @@ <h2 class="topic">
             </ul>
         </div>
     </section>
+
+    <footer id="contact" class="footer">
+        <div class="contact-me">
+            <h2>Contact me</h2>
+            <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+            </p>
+        </div>
+
+        <div class="contact-forms">
+            <form name="form1" class="message-form" action="https://formspree.io/f/xqknrzed" method="post">
+
+                <div class="input-li">
+                    <input type="text" name="text" id="text-input" placeholder="Yeremias" maxlength="30" required>
+                </div>
+
+                <div class="input-li">
+                    <input type="email" name="email" id="mail" placeholder="natajay14@gmail.com" required>
+                </div>
+
+                <div class="text-area-li">
+                    <textarea name="msg" id="msg" maxlength="500" placeholder="Write your messages here"
+                        required></textarea>
+                </div>
+                <div class="submit-li">
+                    <button type="submit" class="project2">Get in Touch</button>
```

Comment:
- [ ] Please 🙏🏽 correct the dimensions of the button to look similar to the Figma template 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/186797155-e97b12d9-d692-4d3c-a0a8-7682cc5b6579.png)

- Your design

![image](https://user-images.githubusercontent.com/58642949/186797180-4ed7ce22-e027-4254-b590-e39b8954c936.png)

Reply:
good job following the previous reviewer's suggestions

Reply:
As you may know I am currently on the contact form and have been able to do the first reviewer modification
the ones being proposed now are different than the first which is not supposed to be the case